pom.xml

```diff
@@ -369,6 +370,45 @@
           </plugin>
         </plugins>
       </build>
+      <dependencies>
+        <dependency>
+          <groupId>org.arquillian.cube</groupId>
+          <artifactId>arquillian-cube-kubernetes-starter</artifactId>
+          <version>${arquillian.cube.version}</version>
+          <scope>test</scope>
+        </dependency>
+      </dependencies>
+    </profile>
+    <profile>
+      <id>itests-openshift</id>
+      <modules>
+        <module>kubernetes-itests</module>
```

Comment:
This module name looks confusing to me. As it is not about k8s or openshift, it's only about `itests` wdyt?

Reply:
if you see, all modules are like this, `kubernetes-modulename`